src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -273,28 +277,33 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
 
                                         break;
                                     }
+                                } else
+                                {
+                                    logDebugBuilder.AppendLine(LoggingConstants.RefreshKeyValueUnchanged + watchedKey);

```

Comment:
We dont need to log unchanged key-values. Logging what changed should be enough.

Reply:
We often hear from customers complaining the config change is not picked up, but the reality is what they are monitoring is different from what they changed. So from debugging perspective, this could be useful.